packages/block-editor/src/components/block-popover/inbetween.js

```diff
@@ -28,6 +34,9 @@ function BlockPopoverInbetween( {
 	__unstableContentRef,
 	...props
 } ) {
+	// This is a temporary hack to get the inbetween inserter to recompute properly.
+	const [ positionRecompute, forceRecompute ] = useState( {} );
```

Comment:
Minor: If resize the window the inserter position becomes wrong.
![image](https://user-images.githubusercontent.com/11271197/170544470-fcfbcaf5-2592-49b8-aeb4-7b32489e3d08.png)

Reply:
This is fixed.